bot/cogs/fun.py

```diff
@@ -14,7 +14,7 @@
 from wand.image import Image
 
 
-from bot.constants import ADMIN_ROLES, EVERYONE_REACTIONS
+from bot.constants import ADMIN_ROLES, EVERYONE_REACTIONS, QUOTES_CHANNEL_ID
```

Comment:
Although the bot is just designed for a single server, having a default constant set feels wrong, somewhat.

Reply:
There's a few different ways it can be addressed:

- Using the command in order to set the bot channel, which would need to be done every time the bot reboots, but allows more flexibility while the bot is running.
- Setting an ID in the constants file, meaning the bot needs no further configuration on boot, but may be less flexible, requiring a restart in order to change the ID.
- Setting a default, constant ID while maintaining the command that could be used to change this ID. This keeps flexibility, but also allows the bot to run without a hitch if the channel is not set.

The production build currently uses the first, whereas this set of changes incorporates the third of these options. There could well be other options worth exploring though.

Reply:
We can't really have persistence with Docker containers, which is the issue.

Reply:
@fractalpenguin I disagree - the `QUOTES_CHANNEL_ID` seems like a good constant to have.
Storing the constant as an environment variable would still require a reboot of the bot, no matter what approach we take.
The only approach that would work with this mindset is calling `os.environ.get` or `os.environ.__getitem__` on each usage of the quotes command. This adds unnecessary overhead, and is a waste of resources compared to simply using a constant defined in `constants.py`. For our use-case, it's safe to say that we'll never change the quotes channel. On the off chance that we do, it's a fairly easy fix to make.

I do agree that these constants should be configurable, so what I could suggest is using a combination of both approaches:
```python
QUOTES_CHANNEL_ID = os.environ.get('QUOTES_CHANNEL_ID', '463657120441696256')
```
This would allow us to optionally override the constant, or fallback to a default value if the appropriate environment variable isn't found. We could also have an optional YAML configuration file, which would override constants defined here.

Reply:
An environment variable can be specified using a .env file, which can be untracked from version control, making the bot server-agnostic.

I do, however, like the environment variable, with a predefined fallback.